homeassistant/components/apcupsd/sensor.py

```diff
@@ -34,392 +34,343 @@
     "alarmdel": SensorEntityDescription(
         key="alarmdel",
         translation_key="alarm_delay",
-        name="UPS Alarm Delay",
     ),
     "ambtemp": SensorEntityDescription(
         key="ambtemp",
-        name="UPS Ambient Temperature",
+        translation_key="ambient_temperature",
         native_unit_of_measurement=UnitOfTemperature.CELSIUS,
         device_class=SensorDeviceClass.TEMPERATURE,
         state_class=SensorStateClass.MEASUREMENT,
     ),
     "apc": SensorEntityDescription(
         key="apc",
         translation_key="apc_status",
-        name="UPS Status Data",
         entity_registry_enabled_default=False,
     ),
     "apcmodel": SensorEntityDescription(
         key="apcmodel",
         translation_key="apc_model",
-        name="UPS Model",
         entity_registry_enabled_default=False,
     ),
```

Comment:
Would it make sense to deprecate this sensor in the long run since this is already exposed by the Device info?

Reply:
Sounds good. We never really tried to remove such redundant sensors. Now that most refactors are done and it is in a better state, we should revisit and deprecated some for sure. Will do that in a follow-up PR.